predicators/nsrt_learning/strips_learning/base_strips_learner.py

```diff
@@ -15,15 +15,13 @@
 class BaseSTRIPSLearner(abc.ABC):
     """Base class definition."""
 
-    def __init__(
-        self,
-        trajectories: List[LowLevelTrajectory],
-        train_tasks: List[Task],
-        predicates: Set[Predicate],
-        segmented_trajs: List[List[Segment]],
-        verify_harmlessness: bool,
-        verbose: bool = True,
-    ) -> None:
+    def __init__(self,
+                 trajectories: List[LowLevelTrajectory],
+                 train_tasks: List[Task],
+                 predicates: Set[Predicate],
+                 segmented_trajs: List[List[Segment]],
+                 verify_harmlessness: bool,
+                 verbose: bool = True) -> None:
```

Comment:
ditto about `yapf`

Reply:
yapf actually forces it back to this change, and idk why lol...